app/index.html

```diff
@@ -0,0 +1,640 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel=stylesheet href="bower_components/bootstrap/dist/css/bootstrap.css">
+    <link rel="stylesheet" href="bower_components/mdi/css/materialdesignicons.min.css">
+    <link rel="stylesheet" href="bower_components/owl.carousel/dist/assets/owl.carousel.min.css">
+    <link rel="stylesheet" href="bower_components/owl.carousel/dist/assets/owl.theme.default.min.css">
+
+    <link href="main.css" rel="stylesheet">
+
+    <title>Shin Onepage Personal Portfolio Preview - ThemeForest</title>
+
+</head>
+<body>
+<div class="page-wrapper">
+
+    <div class="site-spinner"></div>
+
+
+    <header class="navbar navbar-default navbar-fixed-top">
+        <div class="nav-container">
+            <div class="col-md-6">
+                <a href="index.html" class="site-logo">SHIN</a>
+            </div>
+            <div class="col-md-6">
+                <span class="menu-ham" onclick="openNav()">&#9776;</span>
+            </div>
+
+        </div>
+        <div id="mySidenav" class="sidenav">
+            <a href="javascript:void(0)" class="closebtn" onclick="closeNav()">&times;</a>
+
+            <div class="dropdown-inner current" id="navi">
+                <ul class="main-menu mar-bot-2">
+                    <li><a href="#home" class="current">Home</a></li>
+                    <li><a href="#about">About</a></li>
+                    <li><a href="#resume">Resume</a></li>
+                    <li><a href="#latest-work">Work</a></li>
+                    <li><a href="#portfolio">Portfolio</a></li>
+                    <li><a href="#contact">Contact</a></li>
+                </ul>
+
+                <span class="inline text-bold">Fallow Me</span>
+                <div class="social inline">
+                    <a href="" class="social-icon"><i class="mdi mdi-facebook"></i> </a>
```

Comment:
href should not be empty. If you dont want to use it, then you should change selector to e.g. span